doc/nrf/protocols/matter/end_product/certification.rst

```diff
@@ -209,7 +209,7 @@ The following table lists Matter over Thread certification requirements for when
 
 Bluetooth and Thread certifications can be inherited from Nordic Semiconductor (see the :ref:`ug_matter_device_certification_reqs_dependent` section below).
```

Comment:
Would you mind adding registered trademark to first Bluetooth appearance for this page too? I think it might be in the table above, line 205.

Reply:
Added.